src/pages/home/subscription/subscription.styled.js

```diff
@@ -1,10 +1,78 @@
 import styled from 'styled-components';
-import { colors } from '@tidb-community/ui';
+import { colors, mixins } from '@tidb-community/ui';
 
 import { Section } from '~/pages/home/index.styled';
+import { Button, Input, Row } from 'antd';
 
-export const Container = styled(Section)`
-  && {
-    background: ${colors.M2};
+export const Container = styled.div`
+  ${mixins.reset()}
+  ${mixins.centerBlock()}
+  ${mixins.responsive()}
+
+  ${(props) =>
+    props.isMobile &&
+    `
+    text-align: center;
+    ${mixins.flexCenter()}
+  `}
+`;
+
+export const SloganDiv = styled(Row)`
+  max-width: 634px;
+  padding-right: 8px;
+  padding-left: 8px;
+`;
+
+export const SubscriptionSection = styled(Section)`
+  ${mixins.flexCenter()}
+  background-image: url("images/tugHome/subscription-background.svg");
+  background-size: cover;
+  padding-top: 62px;
+  padding-bottom: 62px;
+`;
+
+export const Slogan = styled.div`
+  color: ${colors.M1};
+  font-size: 20px;
+`;
+export const TermCaption = styled.div`
+  margin-top: 36px;
+  color: ${colors.M1};
+  font-size: 14px;
+`;
+
+export const Link = styled.a`
+  color: ${colors.M1};
+  font-size: 20px;
+  text-decoration: underline;
+
+  &:hover {
+    text-decoration: underline;
+    color: inherit;
+    cursor: pointer;
+  }
+`;
+
+export const JoinButton = styled(Button)`
+  ${mixins.flexCenter()}
+  ${(props) => (props.isMobile ? `margin-bottom: 50px; margin-top: 28px` : `margin-top: 46px;`)}
```

Comment:
You may have better code formatting if applying the [css helper](https://styled-components.com/docs/api#css).
Please take a look of https://github.com/pingcap/tug-website/blob/main/src/pages/home/banner/banner.styled.js#L26